packages/subgraph/CHANGELOG.md

```diff
@@ -9,12 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Split
 
-- Renamed Proposal to `TransactionActions` and made it immutable
-- Added deterministicID as field to `TransactionActions` and `TransactionAction`
-- Temp immutable entity `TransactionAction` created while plugins (with actions) still live inside OSx. Includes `deterministicId` and swaps `proposal` foreign key in place of `transactionAction`
+- Renamed Proposal to `ActionBatch` and made it immutable
+- Added `deterministicId` as field to `ActionBatch` and `Action`. The `deterministicId` is a reference shared by the ActionId in plugins packages that can be known at the time of proposal execution.
 - The `deterministicId` and `Id` fields now include the `daoAddress` inside executeActions
-- Renamed `proposal` to `transactionActions` in ERC20Transfer, ERC721Transfer, ERC1155Transfer and NativeTransfer
-- Removed `startDate` and `endDate` from `TransactionActions`
+- Renamed `proposal` to `actionBatch` in `Action`, `ERC20Transfer`, `ERC721Transfer`, `ERC1155Transfer`, `NativeTransfer` and `DAO`
```

Comment:
```suggestion
- Renamed `Proposal` to `actionBatch` in `Action`, `ERC20Transfer`, `ERC721Transfer`, `ERC1155Transfer`, `NativeTransfer` and `DAO`
```

Reply:
Deliberately ignored, the original is correct